SleepTracker/SleepTracker/Helpers/CGFloat+AppFontSize.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+extension CGFloat {
+    /// The font size used in the application.
+    enum AppFontSize {
+        /// A font size used for caption.
```

Comment:
nit:
```suggestion
        /// A font size used for captions.
```

to be consistent with the rest of doc comments.

Reply:
Fixed.